### PR TITLE
Fit/fill contextual menu in video gallery tiles

### PR DESCRIPTION
**Pull request by anjulgarg**
# What
Fit/fill contextual menu in video gallery tiles

https://user-images.githubusercontent.com/9782747/205689769-7dd2dc23-ade5-4e92-b230-508a693134b0.mov

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/RemoteVideoTile.tsx [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
+  VideoGalleryRemoteParticipant,
+  VideoStreamOptions
+} from '../types';
 import { StreamMedia } from './StreamMedia';
+import { VideoGalleryStrings } from './VideoGallery';
 import {
-  useRemoteVideoStreamLifecycleMaintainer,
-  RemoteVideoStreamLifecycleMaintainerProps
+  RemoteVideoStreamLifecycleMaintainerProps,
+  useRemoteVideoStreamLifecycleMaintainer
```
Comment (edwardlee-msft) [inline packages/react-components/src/components/RemoteVideoTile.tsx:17]: Like the alphabetical order change here.

**Review by dmceachernmsft — APPROVED** [review]
Overall looks good, something I am wondering though is why some of these mobile snapshots have participants speaking now? was this something you know about? I don't see the change here 
image

**Comment by anjulgarg** [comment]
> Overall looks good, something I am wondering though is why some of these mobile snapshots have participants speaking now? was this something you know about? I don't see the change here
> image

That is so weird. I tried updating the snapshots again but still shows like this -_-

File: packages/react-components/src/components/RemoteVideoTile.tsx [hunk 2]
```diff
@@ -103,17 +126,32 @@ export const _RemoteVideoTile = React.memo(
...
+): { contextualMenu?: IContextualMenuProps } => {
+  if (!contextualMenuProps) {
+    return {};
+  }
+  /* @conditional-compile-remove(pinned-participants) */
+  return {
+    contextualMenu: contextualMenuProps
+  };
+
+  return {};
```
Comment (edwardlee-msft) [inline packages/react-components/src/components/RemoteVideoTile.tsx:156]: nit: could this be changed to:
```
/* @conditional-compile-remove(pinned-participants) */
if (contextualMenuProps) return {contextualMenu: contextualMenuProps};

return {}
```

**Comment by edwardlee-msft** [comment]
The ellipsis padding on the right seems a bit large but other than that, looks great :)
image

**Review by edwardlee-msft — APPROVED** [review]

**Pull request merged**